doc/active_docs/Account Management API.json

```diff
@@ -5644,6 +5644,41 @@
         }
       ]
     },
+    {
+      "path": "/admin/api/objects/status.json",
+      "operations": [
+        {
+          "httpMethod": "GET",
+          "summary": "Object deletion status for objects that are deleted asynchronously",
+          "description": "Returns an object status. (200/404). Useful for those objects that deleted asynchronously in order to know if the deletion has been completed(404) or not(200)",
+          "group": "objects",
+          "parameters": [
+            {
+              "name": "access_token",
+              "description": "A personal Access Token",
+              "dataType": "string",
+              "required": true,
+              "paramType": "query",
+              "threescale_name": "access_token"
+            },
+            {
+              "dataType": "string",
+              "required": true,
+              "paramType": "query",
+              "name": "object_type",
+              "description": "Object type could be service, account, proxy or backend_api."
```

Comment:
```suggestion
              "description": "Object type has to be service, account, proxy or backend_api."
```

Reply:
The ActiveDocs are autogenerated, so this shouldn't be changed here but in the documentation in the controller :smile: